book/src/design/gadgets/sha256.md

```diff
@@ -56,7 +56,7 @@ pub trait Sha256Instructions: Chip {
 TODO: Add instruction for computing padding.
 
 This set of instructions was chosen to strike a balance between the reusability of the
-instructions, and the scope for chips to internally optimise them. In particular, we
+instructions, and the scope for chips to internally optimize them. In particular, we
```

Comment:
This isn't a typo, it's just the British or New Zealand English spelling. We don't have a preferred English dialect for technical documentation.

Reply:
Thank you for the review.
Sorry I didn't know that.
I made it back to original spelling.